build/Nuget/Microsoft.VisualStudio.ProjectSystem.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.VisualStudio.ProjectSystem</id>
+    <summary>CoreXT package for Microsoft Project System for C# and VB </summary>
```

Comment:
Change CoreXT to something that isn't a internal name, like "Visual Studio Insertion Package or something something"